Build fails on Darwin Kernel  (arch: arm64)

**Describe this problem**

Build CeresDB fails on my MacBook [MacBook Pro (13-inch, M1, 2020)]

**Steps to reproduce**

cargo build and then I will get the compile error, see the additional infos (the error logs)

**Expected behavior**

Compile phase passed, and get the CeresDB binary

**Additional Information**

<details>
  <summary>Detail error log</summary>
  

```
   Compiling fasthash-sys v0.3.2
   Compiling serde v1.0.137
   Compiling thiserror v1.0.31
   Compiling tokio v1.19.2
   Compiling futures-util v0.3.21
   Compiling failure_derive v0.1.8
   Compiling comfy-table v5.0.1
   Compiling multiversion v0.6.1
   Compiling zerocopy-derive v0.2.0
   Compiling tracing v0.1.35
The following warnings were emitted during compilation:

warning: clang: warning: argument unused during compilation: '-msse4.2' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-maes' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-mavx' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-mavx2' [-Wunused-command-line-argument]
warning: In file included from src/fasthash.cpp:1:
warning: In file included from src/fasthash.hpp:5:
warning: src/smhasher/mum.h:119:15: warning: expression result unused [-Wunused-value]
warning:   lo = v * p, hi;
warning:               ^~
warning: 1 warning generated.
warning: clang: warning: argument unused during compilation: '-msse4.2' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-maes' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-mavx' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-mavx2' [-Wunused-command-line-argument]
warning: src/smhasher/City.cpp:148:21: warning: comparison of integers of different signs: 'int' and 'size_t' (aka 'unsigned long') [-Wsign-compare]
warning:   for (int i = 0; i < len; i++) {
warning:                   ~ ^ ~~~
warning: 1 warning generated.
warning: clang: warning: argument unused during compilation: '-msse4.2' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-maes' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-mavx' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-mavx2' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-msse4.2' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-maes' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-mavx' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-mavx2' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-msse4.2' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-maes' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-mavx' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-mavx2' [-Wunused-command-line-argument]
warning: In file included from src/smhasher/mum.cc:1:
warning: src/smhasher/mum.h:119:15: warning: expression result unused [-Wunused-value]
warning:   lo = v * p, hi;
warning:               ^~
warning: 1 warning generated.
warning: clang: warning: argument unused during compilation: '-msse4.2' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-maes' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-mavx' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-mavx2' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-msse4.2' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-maes' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-mavx' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-mavx2' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-msse4.2' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-maes' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-mavx' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-mavx2' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-msse4.2' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-maes' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-mavx' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-mavx2' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-msse4.2' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-maes' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-mavx' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-mavx2' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-msse4.2' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-maes' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-mavx' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-mavx2' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-msse4.2' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-maes' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-mavx' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-mavx2' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-msse4.2' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-maes' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-mavx' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-mavx2' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-msse4.2' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-maes' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-mavx' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-mavx2' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-msse4.2' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-maes' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-mavx' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-mavx2' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-msse4.2' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-maes' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-mavx' [-Wunused-command-line-argument]
warning: clang: warning: argument unused during compilation: '-mavx2' [-Wunused-command-line-argument]
warning: In file included from src/smhasher/metrohash64crc.cpp:28:
warning: In file included from /Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/nmmintrin.h:15:
warning: In file included from /Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/smmintrin.h:13:
warning: In file included from /Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/tmmintrin.h:13:
warning: In file included from /Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/pmmintrin.h:13:
warning: In file included from /Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/emmintrin.h:13:
warning: In file included from /Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/xmmintrin.h:13:
warning: /Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:33:5: error: use of undeclared identifier '__builtin_ia32_emms'; did you mean '__builtin_isless'?
warning:     __builtin_ia32_emms();
warning:     ^
warning: /Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:33:5: note: '__builtin_isless' declared here
warning: /Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:33:25: error: too few arguments to function call, expected 2, have 0
warning:     __builtin_ia32_emms();
warning:                         ^
warning: /Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:50:19: error: use of undeclared identifier '__builtin_ia32_vec_init_v2si'
warning:     return (__m64)__builtin_ia32_vec_init_v2si(__i, 0);
warning:                   ^
warning: /Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:67:12: error: use of undeclared identifier '__builtin_ia32_vec_ext_v2si'
warning:     return __builtin_ia32_vec_ext_v2si((__v2si)__m, 0);
warning:            ^
warning: /Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:129:19: error: use of undeclared identifier '__builtin_ia32_packsswb'
warning:     return (__m64)__builtin_ia32_packsswb((__v4hi)__m1, (__v4hi)__m2);
warning:                   ^
warning: /Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:159:19: error: use of undeclared identifier '__builtin_ia32_packssdw'
warning:     return (__m64)__builtin_ia32_packssdw((__v2si)__m1, (__v2si)__m2);
warning:                   ^
warning: /Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:189:19: error: use of undeclared identifier '__builtin_ia32_packuswb'
warning:     return (__m64)__builtin_ia32_packuswb((__v4hi)__m1, (__v4hi)__m2);
warning:                   ^
warning: /Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:216:19: error: use of undeclared identifier '__builtin_ia32_punpckhbw'
warning:     return (__m64)__builtin_ia32_punpckhbw((__v8qi)__m1, (__v8qi)__m2);
warning:                   ^
warning: /Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:239:19: error: use of undeclared identifier '__builtin_ia32_punpckhwd'
warning:     return (__m64)__builtin_ia32_punpckhwd((__v4hi)__m1, (__v4hi)__m2);
warning:                   ^
warning: /Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:260:19: error: use of undeclared identifier '__builtin_ia32_punpckhdq'
warning:     return (__m64)__builtin_ia32_punpckhdq((__v2si)__m1, (__v2si)__m2);
warning:                   ^
warning: /Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:287:19: error: use of undeclared identifier '__builtin_ia32_punpcklbw'
warning:     return (__m64)__builtin_ia32_punpcklbw((__v8qi)__m1, (__v8qi)__m2);
warning:                   ^
warning: /Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:310:19: error: use of undeclared identifier '__builtin_ia32_punpcklwd'
warning:     return (__m64)__builtin_ia32_punpcklwd((__v4hi)__m1, (__v4hi)__m2);
warning:                   ^
warning: /Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:331:19: error: use of undeclared identifier '__builtin_ia32_punpckldq'
warning:     return (__m64)__builtin_ia32_punpckldq((__v2si)__m1, (__v2si)__m2);
warning:                   ^
warning: /Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:352:19: error: use of undeclared identifier '__builtin_ia32_paddb'; did you mean '__builtin_arm_addg'?
warning:     return (__m64)__builtin_ia32_paddb((__v8qi)__m1, (__v8qi)__m2);
warning:                   ^
warning: /Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:352:19: note: '__builtin_arm_addg' declared here
warning: /Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:352:19: error: first argument of MTE builtin function must be a pointer ('__v8qi' (vector of 8 'char' values) invalid)
warning:     return (__m64)__builtin_ia32_paddb((__v8qi)__m1, (__v8qi)__m2);
warning:                   ^                    ~~~~~~~~~~~~
warning: /Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:373:19: error: use of undeclared identifier '__builtin_ia32_paddw'; did you mean '__builtin_arm_addg'?
warning:     return (__m64)__builtin_ia32_paddw((__v4hi)__m1, (__v4hi)__m2);
warning:                   ^
warning: /Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:352:19: note: '__builtin_arm_addg' declared here
warning:     return (__m64)__builtin_ia32_paddb((__v8qi)__m1, (__v8qi)__m2);
warning:                   ^
warning: /Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:373:19: error: first argument of MTE builtin function must be a pointer ('__v4hi' (vector of 4 'short' values) invalid)
warning:     return (__m64)__builtin_ia32_paddw((__v4hi)__m1, (__v4hi)__m2);
warning:                   ^                    ~~~~~~~~~~~~
warning: /Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:394:19: error: use of undeclared identifier '__builtin_ia32_paddd'; did you mean '__builtin_arm_addg'?
warning:     return (__m64)__builtin_ia32_paddd((__v2si)__m1, (__v2si)__m2);
warning:                   ^
warning: /Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:352:19: note: '__builtin_arm_addg' declared here
warning:     return (__m64)__builtin_ia32_paddb((__v8qi)__m1, (__v8qi)__m2);
warning:                   ^
warning: /Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:394:19: error: first argument of MTE builtin function must be a pointer ('__v2si' (vector of 2 'int' values) invalid)
warning:     return (__m64)__builtin_ia32_paddd((__v2si)__m1, (__v2si)__m2);
warning:                   ^                    ~~~~~~~~~~~~
warning: fatal error: too many errors emitted, stopping now [-ferror-limit=]
warning: 20 errors generated.

error: failed to run custom build command for `fasthash-sys v0.3.2`

Caused by:
  process didn't exit successfully: `/Users/chenwr/workspace/ceresdb/target/debug/build/fasthash-sys-1e6c4b44a7398dc5/build-script-build` (exit status: 101)
  --- stdout
  TARGET = Some("aarch64-apple-darwin")
  OPT_LEVEL = Some("0")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  CC_aarch64-apple-darwin = None
  CC_aarch64_apple_darwin = None
  HOST_CC = None
  CC = None
  HOST = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  CFLAGS_aarch64-apple-darwin = None
  CFLAGS_aarch64_apple_darwin = None
  HOST_CFLAGS = None
  CFLAGS = None
  DEBUG = Some("true")
  running: "cc" "-O0" "-ffunction-sections" "-fdata-sections" "-fPIC" "-g" "-Wno-implicit-fallthrough" "-Wno-unknown-attributes" "-msse4.2" "-maes" "-mavx" "-mavx2" "-DT1HA0_RUNTIME_SELECT=1" "-DT1HA0_AESNI_AVAILABLE=1" "-Wall" "-Wextra" "-o" "/Users/chenwr/workspace/ceresdb/target/debug/build/fasthash-sys-bf79a6c7afd57249/out/src/fasthash.o" "-c" "src/fasthash.cpp"
  cargo:warning=clang: warning: argument unused during compilation: '-msse4.2' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-maes' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-mavx' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-mavx2' [-Wunused-command-line-argument]
  cargo:warning=In file included from src/fasthash.cpp:1:
  cargo:warning=In file included from src/fasthash.hpp:5:
  cargo:warning=src/smhasher/mum.h:119:15: warning: expression result unused [-Wunused-value]
  cargo:warning=  lo = v * p, hi;
  cargo:warning=              ^~
  cargo:warning=1 warning generated.
  exit status: 0
  TARGET = Some("aarch64-apple-darwin")
  OPT_LEVEL = Some("0")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  CC_aarch64-apple-darwin = None
  CC_aarch64_apple_darwin = None
  HOST_CC = None
  CC = None
  HOST = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  CFLAGS_aarch64-apple-darwin = None
  CFLAGS_aarch64_apple_darwin = None
  HOST_CFLAGS = None
  CFLAGS = None
  DEBUG = Some("true")
  running: "cc" "-O0" "-ffunction-sections" "-fdata-sections" "-fPIC" "-g" "-Wno-implicit-fallthrough" "-Wno-unknown-attributes" "-msse4.2" "-maes" "-mavx" "-mavx2" "-DT1HA0_RUNTIME_SELECT=1" "-DT1HA0_AESNI_AVAILABLE=1" "-Wall" "-Wextra" "-o" "/Users/chenwr/workspace/ceresdb/target/debug/build/fasthash-sys-bf79a6c7afd57249/out/src/smhasher/City.o" "-c" "src/smhasher/City.cpp"
  cargo:warning=clang: warning: argument unused during compilation: '-msse4.2' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-maes' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-mavx' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-mavx2' [-Wunused-command-line-argument]
  cargo:warning=src/smhasher/City.cpp:148:21: warning: comparison of integers of different signs: 'int' and 'size_t' (aka 'unsigned long') [-Wsign-compare]
  cargo:warning=  for (int i = 0; i < len; i++) {
  cargo:warning=                  ~ ^ ~~~
  cargo:warning=1 warning generated.
  exit status: 0
  TARGET = Some("aarch64-apple-darwin")
  OPT_LEVEL = Some("0")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  CC_aarch64-apple-darwin = None
  CC_aarch64_apple_darwin = None
  HOST_CC = None
  CC = None
  HOST = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  CFLAGS_aarch64-apple-darwin = None
  CFLAGS_aarch64_apple_darwin = None
  HOST_CFLAGS = None
  CFLAGS = None
  DEBUG = Some("true")
  running: "cc" "-O0" "-ffunction-sections" "-fdata-sections" "-fPIC" "-g" "-Wno-implicit-fallthrough" "-Wno-unknown-attributes" "-msse4.2" "-maes" "-mavx" "-mavx2" "-DT1HA0_RUNTIME_SELECT=1" "-DT1HA0_AESNI_AVAILABLE=1" "-Wall" "-Wextra" "-o" "/Users/chenwr/workspace/ceresdb/target/debug/build/fasthash-sys-bf79a6c7afd57249/out/src/smhasher/farmhash-c.o" "-c" "src/smhasher/farmhash-c.c"
  cargo:warning=clang: warning: argument unused during compilation: '-msse4.2' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-maes' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-mavx' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-mavx2' [-Wunused-command-line-argument]
  exit status: 0
  TARGET = Some("aarch64-apple-darwin")
  OPT_LEVEL = Some("0")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  CC_aarch64-apple-darwin = None
  CC_aarch64_apple_darwin = None
  HOST_CC = None
  CC = None
  HOST = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  CFLAGS_aarch64-apple-darwin = None
  CFLAGS_aarch64_apple_darwin = None
  HOST_CFLAGS = None
  CFLAGS = None
  DEBUG = Some("true")
  running: "cc" "-O0" "-ffunction-sections" "-fdata-sections" "-fPIC" "-g" "-Wno-implicit-fallthrough" "-Wno-unknown-attributes" "-msse4.2" "-maes" "-mavx" "-mavx2" "-DT1HA0_RUNTIME_SELECT=1" "-DT1HA0_AESNI_AVAILABLE=1" "-Wall" "-Wextra" "-o" "/Users/chenwr/workspace/ceresdb/target/debug/build/fasthash-sys-bf79a6c7afd57249/out/src/smhasher/lookup3.o" "-c" "src/smhasher/lookup3.cpp"
  cargo:warning=clang: warning: argument unused during compilation: '-msse4.2' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-maes' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-mavx' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-mavx2' [-Wunused-command-line-argument]
  exit status: 0
  TARGET = Some("aarch64-apple-darwin")
  OPT_LEVEL = Some("0")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  CC_aarch64-apple-darwin = None
  CC_aarch64_apple_darwin = None
  HOST_CC = None
  CC = None
  HOST = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  CFLAGS_aarch64-apple-darwin = None
  CFLAGS_aarch64_apple_darwin = None
  HOST_CFLAGS = None
  CFLAGS = None
  DEBUG = Some("true")
  running: "cc" "-O0" "-ffunction-sections" "-fdata-sections" "-fPIC" "-g" "-Wno-implicit-fallthrough" "-Wno-unknown-attributes" "-msse4.2" "-maes" "-mavx" "-mavx2" "-DT1HA0_RUNTIME_SELECT=1" "-DT1HA0_AESNI_AVAILABLE=1" "-Wall" "-Wextra" "-o" "/Users/chenwr/workspace/ceresdb/target/debug/build/fasthash-sys-bf79a6c7afd57249/out/src/smhasher/mum.o" "-c" "src/smhasher/mum.cc"
  cargo:warning=clang: warning: argument unused during compilation: '-msse4.2' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-maes' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-mavx' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-mavx2' [-Wunused-command-line-argument]
  cargo:warning=In file included from src/smhasher/mum.cc:1:
  cargo:warning=src/smhasher/mum.h:119:15: warning: expression result unused [-Wunused-value]
  cargo:warning=  lo = v * p, hi;
  cargo:warning=              ^~
  cargo:warning=1 warning generated.
  exit status: 0
  TARGET = Some("aarch64-apple-darwin")
  OPT_LEVEL = Some("0")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  CC_aarch64-apple-darwin = None
  CC_aarch64_apple_darwin = None
  HOST_CC = None
  CC = None
  HOST = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  CFLAGS_aarch64-apple-darwin = None
  CFLAGS_aarch64_apple_darwin = None
  HOST_CFLAGS = None
  CFLAGS = None
  DEBUG = Some("true")
  running: "cc" "-O0" "-ffunction-sections" "-fdata-sections" "-fPIC" "-g" "-Wno-implicit-fallthrough" "-Wno-unknown-attributes" "-msse4.2" "-maes" "-mavx" "-mavx2" "-DT1HA0_RUNTIME_SELECT=1" "-DT1HA0_AESNI_AVAILABLE=1" "-Wall" "-Wextra" "-o" "/Users/chenwr/workspace/ceresdb/target/debug/build/fasthash-sys-bf79a6c7afd57249/out/src/smhasher/metrohash64.o" "-c" "src/smhasher/metrohash64.cpp"
  cargo:warning=clang: warning: argument unused during compilation: '-msse4.2' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-maes' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-mavx' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-mavx2' [-Wunused-command-line-argument]
  exit status: 0
  TARGET = Some("aarch64-apple-darwin")
  OPT_LEVEL = Some("0")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  CC_aarch64-apple-darwin = None
  CC_aarch64_apple_darwin = None
  HOST_CC = None
  CC = None
  HOST = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  CFLAGS_aarch64-apple-darwin = None
  CFLAGS_aarch64_apple_darwin = None
  HOST_CFLAGS = None
  CFLAGS = None
  DEBUG = Some("true")
  running: "cc" "-O0" "-ffunction-sections" "-fdata-sections" "-fPIC" "-g" "-Wno-implicit-fallthrough" "-Wno-unknown-attributes" "-msse4.2" "-maes" "-mavx" "-mavx2" "-DT1HA0_RUNTIME_SELECT=1" "-DT1HA0_AESNI_AVAILABLE=1" "-Wall" "-Wextra" "-o" "/Users/chenwr/workspace/ceresdb/target/debug/build/fasthash-sys-bf79a6c7afd57249/out/src/smhasher/metrohash128.o" "-c" "src/smhasher/metrohash128.cpp"
  cargo:warning=clang: warning: argument unused during compilation: '-msse4.2' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-maes' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-mavx' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-mavx2' [-Wunused-command-line-argument]
  exit status: 0
  TARGET = Some("aarch64-apple-darwin")
  OPT_LEVEL = Some("0")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  CC_aarch64-apple-darwin = None
  CC_aarch64_apple_darwin = None
  HOST_CC = None
  CC = None
  HOST = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  CFLAGS_aarch64-apple-darwin = None
  CFLAGS_aarch64_apple_darwin = None
  HOST_CFLAGS = None
  CFLAGS = None
  DEBUG = Some("true")
  running: "cc" "-O0" "-ffunction-sections" "-fdata-sections" "-fPIC" "-g" "-Wno-implicit-fallthrough" "-Wno-unknown-attributes" "-msse4.2" "-maes" "-mavx" "-mavx2" "-DT1HA0_RUNTIME_SELECT=1" "-DT1HA0_AESNI_AVAILABLE=1" "-Wall" "-Wextra" "-o" "/Users/chenwr/workspace/ceresdb/target/debug/build/fasthash-sys-bf79a6c7afd57249/out/src/smhasher/MurmurHash1.o" "-c" "src/smhasher/MurmurHash1.cpp"
  cargo:warning=clang: warning: argument unused during compilation: '-msse4.2' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-maes' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-mavx' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-mavx2' [-Wunused-command-line-argument]
  exit status: 0
  TARGET = Some("aarch64-apple-darwin")
  OPT_LEVEL = Some("0")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  CC_aarch64-apple-darwin = None
  CC_aarch64_apple_darwin = None
  HOST_CC = None
  CC = None
  HOST = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  CFLAGS_aarch64-apple-darwin = None
  CFLAGS_aarch64_apple_darwin = None
  HOST_CFLAGS = None
  CFLAGS = None
  DEBUG = Some("true")
  running: "cc" "-O0" "-ffunction-sections" "-fdata-sections" "-fPIC" "-g" "-Wno-implicit-fallthrough" "-Wno-unknown-attributes" "-msse4.2" "-maes" "-mavx" "-mavx2" "-DT1HA0_RUNTIME_SELECT=1" "-DT1HA0_AESNI_AVAILABLE=1" "-Wall" "-Wextra" "-o" "/Users/chenwr/workspace/ceresdb/target/debug/build/fasthash-sys-bf79a6c7afd57249/out/src/smhasher/MurmurHash2.o" "-c" "src/smhasher/MurmurHash2.cpp"
  cargo:warning=clang: warning: argument unused during compilation: '-msse4.2' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-maes' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-mavx' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-mavx2' [-Wunused-command-line-argument]
  exit status: 0
  TARGET = Some("aarch64-apple-darwin")
  OPT_LEVEL = Some("0")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  CC_aarch64-apple-darwin = None
  CC_aarch64_apple_darwin = None
  HOST_CC = None
  CC = None
  HOST = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  CFLAGS_aarch64-apple-darwin = None
  CFLAGS_aarch64_apple_darwin = None
  HOST_CFLAGS = None
  CFLAGS = None
  DEBUG = Some("true")
  running: "cc" "-O0" "-ffunction-sections" "-fdata-sections" "-fPIC" "-g" "-Wno-implicit-fallthrough" "-Wno-unknown-attributes" "-msse4.2" "-maes" "-mavx" "-mavx2" "-DT1HA0_RUNTIME_SELECT=1" "-DT1HA0_AESNI_AVAILABLE=1" "-Wall" "-Wextra" "-o" "/Users/chenwr/workspace/ceresdb/target/debug/build/fasthash-sys-bf79a6c7afd57249/out/src/smhasher/MurmurHash3.o" "-c" "src/smhasher/MurmurHash3.cpp"
  cargo:warning=clang: warning: argument unused during compilation: '-msse4.2' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-maes' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-mavx' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-mavx2' [-Wunused-command-line-argument]
  exit status: 0
  TARGET = Some("aarch64-apple-darwin")
  OPT_LEVEL = Some("0")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  CC_aarch64-apple-darwin = None
  CC_aarch64_apple_darwin = None
  HOST_CC = None
  CC = None
  HOST = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  CFLAGS_aarch64-apple-darwin = None
  CFLAGS_aarch64_apple_darwin = None
  HOST_CFLAGS = None
  CFLAGS = None
  DEBUG = Some("true")
  running: "cc" "-O0" "-ffunction-sections" "-fdata-sections" "-fPIC" "-g" "-Wno-implicit-fallthrough" "-Wno-unknown-attributes" "-msse4.2" "-maes" "-mavx" "-mavx2" "-DT1HA0_RUNTIME_SELECT=1" "-DT1HA0_AESNI_AVAILABLE=1" "-Wall" "-Wextra" "-o" "/Users/chenwr/workspace/ceresdb/target/debug/build/fasthash-sys-bf79a6c7afd57249/out/src/smhasher/Spooky.o" "-c" "src/smhasher/Spooky.cpp"
  cargo:warning=clang: warning: argument unused during compilation: '-msse4.2' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-maes' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-mavx' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-mavx2' [-Wunused-command-line-argument]
  exit status: 0
  TARGET = Some("aarch64-apple-darwin")
  OPT_LEVEL = Some("0")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  CC_aarch64-apple-darwin = None
  CC_aarch64_apple_darwin = None
  HOST_CC = None
  CC = None
  HOST = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  CFLAGS_aarch64-apple-darwin = None
  CFLAGS_aarch64_apple_darwin = None
  HOST_CFLAGS = None
  CFLAGS = None
  DEBUG = Some("true")
  running: "cc" "-O0" "-ffunction-sections" "-fdata-sections" "-fPIC" "-g" "-Wno-implicit-fallthrough" "-Wno-unknown-attributes" "-msse4.2" "-maes" "-mavx" "-mavx2" "-DT1HA0_RUNTIME_SELECT=1" "-DT1HA0_AESNI_AVAILABLE=1" "-Wall" "-Wextra" "-o" "/Users/chenwr/workspace/ceresdb/target/debug/build/fasthash-sys-bf79a6c7afd57249/out/src/smhasher/xxhash.o" "-c" "src/smhasher/xxhash.c"
  cargo:warning=clang: warning: argument unused during compilation: '-msse4.2' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-maes' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-mavx' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-mavx2' [-Wunused-command-line-argument]
  exit status: 0
  TARGET = Some("aarch64-apple-darwin")
  OPT_LEVEL = Some("0")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  CC_aarch64-apple-darwin = None
  CC_aarch64_apple_darwin = None
  HOST_CC = None
  CC = None
  HOST = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  CFLAGS_aarch64-apple-darwin = None
  CFLAGS_aarch64_apple_darwin = None
  HOST_CFLAGS = None
  CFLAGS = None
  DEBUG = Some("true")
  running: "cc" "-O0" "-ffunction-sections" "-fdata-sections" "-fPIC" "-g" "-Wno-implicit-fallthrough" "-Wno-unknown-attributes" "-msse4.2" "-maes" "-mavx" "-mavx2" "-DT1HA0_RUNTIME_SELECT=1" "-DT1HA0_AESNI_AVAILABLE=1" "-Wall" "-Wextra" "-o" "/Users/chenwr/workspace/ceresdb/target/debug/build/fasthash-sys-bf79a6c7afd57249/out/src/t1ha/src/t1ha0.o" "-c" "src/t1ha/src/t1ha0.c"
  cargo:warning=clang: warning: argument unused during compilation: '-msse4.2' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-maes' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-mavx' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-mavx2' [-Wunused-command-line-argument]
  exit status: 0
  TARGET = Some("aarch64-apple-darwin")
  OPT_LEVEL = Some("0")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  CC_aarch64-apple-darwin = None
  CC_aarch64_apple_darwin = None
  HOST_CC = None
  CC = None
  HOST = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  CFLAGS_aarch64-apple-darwin = None
  CFLAGS_aarch64_apple_darwin = None
  HOST_CFLAGS = None
  CFLAGS = None
  DEBUG = Some("true")
  running: "cc" "-O0" "-ffunction-sections" "-fdata-sections" "-fPIC" "-g" "-Wno-implicit-fallthrough" "-Wno-unknown-attributes" "-msse4.2" "-maes" "-mavx" "-mavx2" "-DT1HA0_RUNTIME_SELECT=1" "-DT1HA0_AESNI_AVAILABLE=1" "-Wall" "-Wextra" "-o" "/Users/chenwr/workspace/ceresdb/target/debug/build/fasthash-sys-bf79a6c7afd57249/out/src/t1ha/src/t1ha1.o" "-c" "src/t1ha/src/t1ha1.c"
  cargo:warning=clang: warning: argument unused during compilation: '-msse4.2' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-maes' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-mavx' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-mavx2' [-Wunused-command-line-argument]
  exit status: 0
  TARGET = Some("aarch64-apple-darwin")
  OPT_LEVEL = Some("0")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  CC_aarch64-apple-darwin = None
  CC_aarch64_apple_darwin = None
  HOST_CC = None
  CC = None
  HOST = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  CFLAGS_aarch64-apple-darwin = None
  CFLAGS_aarch64_apple_darwin = None
  HOST_CFLAGS = None
  CFLAGS = None
  DEBUG = Some("true")
  running: "cc" "-O0" "-ffunction-sections" "-fdata-sections" "-fPIC" "-g" "-Wno-implicit-fallthrough" "-Wno-unknown-attributes" "-msse4.2" "-maes" "-mavx" "-mavx2" "-DT1HA0_RUNTIME_SELECT=1" "-DT1HA0_AESNI_AVAILABLE=1" "-Wall" "-Wextra" "-o" "/Users/chenwr/workspace/ceresdb/target/debug/build/fasthash-sys-bf79a6c7afd57249/out/src/t1ha/src/t1ha2.o" "-c" "src/t1ha/src/t1ha2.c"
  cargo:warning=clang: warning: argument unused during compilation: '-msse4.2' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-maes' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-mavx' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-mavx2' [-Wunused-command-line-argument]
  exit status: 0
  TARGET = Some("aarch64-apple-darwin")
  OPT_LEVEL = Some("0")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  CC_aarch64-apple-darwin = None
  CC_aarch64_apple_darwin = None
  HOST_CC = None
  CC = None
  HOST = Some("aarch64-apple-darwin")
  TARGET = Some("aarch64-apple-darwin")
  HOST = Some("aarch64-apple-darwin")
  CFLAGS_aarch64-apple-darwin = None
  CFLAGS_aarch64_apple_darwin = None
  HOST_CFLAGS = None
  CFLAGS = None
  DEBUG = Some("true")
  running: "cc" "-O0" "-ffunction-sections" "-fdata-sections" "-fPIC" "-g" "-Wno-implicit-fallthrough" "-Wno-unknown-attributes" "-msse4.2" "-maes" "-mavx" "-mavx2" "-DT1HA0_RUNTIME_SELECT=1" "-DT1HA0_AESNI_AVAILABLE=1" "-Wall" "-Wextra" "-o" "/Users/chenwr/workspace/ceresdb/target/debug/build/fasthash-sys-bf79a6c7afd57249/out/src/smhasher/metrohash64crc.o" "-c" "src/smhasher/metrohash64crc.cpp"
  cargo:warning=clang: warning: argument unused during compilation: '-msse4.2' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-maes' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-mavx' [-Wunused-command-line-argument]
  cargo:warning=clang: warning: argument unused during compilation: '-mavx2' [-Wunused-command-line-argument]
  cargo:warning=In file included from src/smhasher/metrohash64crc.cpp:28:
  cargo:warning=In file included from /Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/nmmintrin.h:15:
  cargo:warning=In file included from /Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/smmintrin.h:13:
  cargo:warning=In file included from /Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/tmmintrin.h:13:
  cargo:warning=In file included from /Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/pmmintrin.h:13:
  cargo:warning=In file included from /Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/emmintrin.h:13:
  cargo:warning=In file included from /Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/xmmintrin.h:13:
  cargo:warning=/Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:33:5: error: use of undeclared identifier '__builtin_ia32_emms'; did you mean '__builtin_isless'?
  cargo:warning=    __builtin_ia32_emms();
  cargo:warning=    ^
  cargo:warning=/Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:33:5: note: '__builtin_isless' declared here
  cargo:warning=/Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:33:25: error: too few arguments to function call, expected 2, have 0
  cargo:warning=    __builtin_ia32_emms();
  cargo:warning=                        ^
  cargo:warning=/Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:50:19: error: use of undeclared identifier '__builtin_ia32_vec_init_v2si'
  cargo:warning=    return (__m64)__builtin_ia32_vec_init_v2si(__i, 0);
  cargo:warning=                  ^
  cargo:warning=/Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:67:12: error: use of undeclared identifier '__builtin_ia32_vec_ext_v2si'
  cargo:warning=    return __builtin_ia32_vec_ext_v2si((__v2si)__m, 0);
  cargo:warning=           ^
  cargo:warning=/Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:129:19: error: use of undeclared identifier '__builtin_ia32_packsswb'
  cargo:warning=    return (__m64)__builtin_ia32_packsswb((__v4hi)__m1, (__v4hi)__m2);
  cargo:warning=                  ^
  cargo:warning=/Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:159:19: error: use of undeclared identifier '__builtin_ia32_packssdw'
  cargo:warning=    return (__m64)__builtin_ia32_packssdw((__v2si)__m1, (__v2si)__m2);
  cargo:warning=                  ^
  cargo:warning=/Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:189:19: error: use of undeclared identifier '__builtin_ia32_packuswb'
  cargo:warning=    return (__m64)__builtin_ia32_packuswb((__v4hi)__m1, (__v4hi)__m2);
  cargo:warning=                  ^
  cargo:warning=/Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:216:19: error: use of undeclared identifier '__builtin_ia32_punpckhbw'
  cargo:warning=    return (__m64)__builtin_ia32_punpckhbw((__v8qi)__m1, (__v8qi)__m2);
  cargo:warning=                  ^
  cargo:warning=/Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:239:19: error: use of undeclared identifier '__builtin_ia32_punpckhwd'
  cargo:warning=    return (__m64)__builtin_ia32_punpckhwd((__v4hi)__m1, (__v4hi)__m2);
  cargo:warning=                  ^
  cargo:warning=/Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:260:19: error: use of undeclared identifier '__builtin_ia32_punpckhdq'
  cargo:warning=    return (__m64)__builtin_ia32_punpckhdq((__v2si)__m1, (__v2si)__m2);
  cargo:warning=                  ^
  cargo:warning=/Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:287:19: error: use of undeclared identifier '__builtin_ia32_punpcklbw'
  cargo:warning=    return (__m64)__builtin_ia32_punpcklbw((__v8qi)__m1, (__v8qi)__m2);
  cargo:warning=                  ^
  cargo:warning=/Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:310:19: error: use of undeclared identifier '__builtin_ia32_punpcklwd'
  cargo:warning=    return (__m64)__builtin_ia32_punpcklwd((__v4hi)__m1, (__v4hi)__m2);
  cargo:warning=                  ^
  cargo:warning=/Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:331:19: error: use of undeclared identifier '__builtin_ia32_punpckldq'
  cargo:warning=    return (__m64)__builtin_ia32_punpckldq((__v2si)__m1, (__v2si)__m2);
  cargo:warning=                  ^
  cargo:warning=/Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:352:19: error: use of undeclared identifier '__builtin_ia32_paddb'; did you mean '__builtin_arm_addg'?
  cargo:warning=    return (__m64)__builtin_ia32_paddb((__v8qi)__m1, (__v8qi)__m2);
  cargo:warning=                  ^
  cargo:warning=/Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:352:19: note: '__builtin_arm_addg' declared here
  cargo:warning=/Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:352:19: error: first argument of MTE builtin function must be a pointer ('__v8qi' (vector of 8 'char' values) invalid)
  cargo:warning=    return (__m64)__builtin_ia32_paddb((__v8qi)__m1, (__v8qi)__m2);
  cargo:warning=                  ^                    ~~~~~~~~~~~~
  cargo:warning=/Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:373:19: error: use of undeclared identifier '__builtin_ia32_paddw'; did you mean '__builtin_arm_addg'?
  cargo:warning=    return (__m64)__builtin_ia32_paddw((__v4hi)__m1, (__v4hi)__m2);
  cargo:warning=                  ^
  cargo:warning=/Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:352:19: note: '__builtin_arm_addg' declared here
  cargo:warning=    return (__m64)__builtin_ia32_paddb((__v8qi)__m1, (__v8qi)__m2);
  cargo:warning=                  ^
  cargo:warning=/Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:373:19: error: first argument of MTE builtin function must be a pointer ('__v4hi' (vector of 4 'short' values) invalid)
  cargo:warning=    return (__m64)__builtin_ia32_paddw((__v4hi)__m1, (__v4hi)__m2);
  cargo:warning=                  ^                    ~~~~~~~~~~~~
  cargo:warning=/Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:394:19: error: use of undeclared identifier '__builtin_ia32_paddd'; did you mean '__builtin_arm_addg'?
  cargo:warning=    return (__m64)__builtin_ia32_paddd((__v2si)__m1, (__v2si)__m2);
  cargo:warning=                  ^
  cargo:warning=/Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:352:19: note: '__builtin_arm_addg' declared here
  cargo:warning=    return (__m64)__builtin_ia32_paddb((__v8qi)__m1, (__v8qi)__m2);
  cargo:warning=                  ^
  cargo:warning=/Library/Developer/CommandLineTools/usr/lib/clang/12.0.5/include/mmintrin.h:394:19: error: first argument of MTE builtin function must be a pointer ('__v2si' (vector of 2 'int' values) invalid)
  cargo:warning=    return (__m64)__builtin_ia32_paddd((__v2si)__m1, (__v2si)__m2);
  cargo:warning=                  ^                    ~~~~~~~~~~~~
  cargo:warning=fatal error: too many errors emitted, stopping now [-ferror-limit=]
  cargo:warning=20 errors generated.
  exit status: 1

  --- stderr
  thread 'main' panicked at '

  Internal error occurred: Command "cc" "-O0" "-ffunction-sections" "-fdata-sections" "-fPIC" "-g" "-Wno-implicit-fallthrough" "-Wno-unknown-attributes" "-msse4.2" "-maes" "-mavx" "-mavx2" "-DT1HA0_RUNTIME_SELECT=1" "-DT1HA0_AESNI_AVAILABLE=1" "-Wall" "-Wextra" "-o" "/Users/chenwr/workspace/ceresdb/target/debug/build/fasthash-sys-bf79a6c7afd57249/out/src/smhasher/metrohash64crc.o" "-c" "src/smhasher/metrohash64crc.cpp" with args "cc" did not execute successfully (status code exit status: 1).

  ', /Users/chenwr/.cargo/registry/src/github.com-1ecc6299db9ec823/gcc-0.3.55/src/lib.rs:1672:5
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
error: build failed
```



</details>

--- a/components/table_kv/Cargo.toml
+++ b/components/table_kv/Cargo.toml
@@ -8,7 +8,7 @@ edition = "2021"
 [dependencies]
 common_util = { path = "../../common_util" }
 log = "0.4"
-obkv-table-client-rs = { git = "https://github.com/oceanbase/obkv-table-client-rs.git", rev = "d525c6d7d8919d98be9a2590bbfa8c417dda6584" }
+obkv-table-client-rs = { git = "https://github.com/oceanbase/obkv-table-client-rs.git", rev = "318907f35de51404c05180c65440c9516fd1e10f" }
 serde = "1.0"
 serde_derive = "1.0"
 snafu = { version = "0.6.10", features = ["backtraces"] }

